RELEASE/scripts/autoscend.ash

```diff
@@ -3292,10 +3288,6 @@ boolean doTasks()
 		{
 			set_property("auto_cubeItems", false);
 		}
-		if(get_property("auto_cubeItems").to_boolean() && (my_familiar() != $familiar[Gelatinous Cubeling]) && auto_have_familiar($familiar[Gelatinous Cubeling]))
```

Comment:
This needs to be in. It is how we get the 3 items that cubeling drops that lets us do daily dungeon quickly